chain/src/main/scala/org/bitcoins/chain/config/ChainAppConfig.scala

```diff
@@ -53,9 +58,14 @@ case class ChainAppConfig(val confs: Config*) extends AppConfig {
           BlockHeaderDbHelper.fromBlockHeader(height = 0,
                                               bh =
                                                 chain.genesisBlock.blockHeader)
+        val blockHeaderDAO =
+          BlockHeaderDAO()(ec = implicitly[ExecutionContext], appConfig = this)
```

Comment:
Just because I wasn't sure what the current execution context in scope was called. `implictly` works at compile level, so shoudln't make a difference. You could also argue that it's more readable, because it operates on the type level, and not on the "whatever I've called my constants"-level. If you disagree I can change it, not a big deal for me.